server/src/main/java/com/broadcom/lsp/cobol/service/CobolLanguageServer.java

```diff
@@ -124,14 +125,21 @@ private void getLocaleFromClient() {
     settingsService.getConfiguration(LOCALE.label).thenAccept(localeStore.notifyLocaleStore());
   }
 
+  public int getExitCode() {
+    return exitCode;
+  }
+
   @Override
   public CompletableFuture<Object> shutdown() {
+    LOG.info("LS for COBOL received shutdown request");
```

Comment:
As per LSP spec, after this call no other service call can/should be handled. Should we have a logic to return no-valid from all our service after this call is made. 
Why I didn't implement it? 
VScode actually never waits for us to call exit. Upon closing, it fires a shutdown call and then fires an exit call.
Details mentioned in the PR comment.

Reply:
We support different IDEs, should it would be better not to rely on the client implementation